apps/namadillo/src/App/AccountOverview/NamBalanceContainer.tsx

```diff
@@ -102,66 +108,106 @@ export const NamBalanceContainer = (): JSX.Element => {
-              color={colors.shielded}
-              amount={maspEnabled ? shieldedNamAmount : new BigNumber(0)}
-              isLoading={shieldedAmountQuery.isLoading}
-              isSyncing={isShieldSyncing}
-              isEnabled={maspEnabled}
-            />
-            <ListItemContainer
-              isEnabled={shieldingRewardsEnabled}
-              className="flex flex-1"
+          <>
+            <Stack
+              as="ul"
+              gap={2}
+              className={clsx("w-full", {
+                "order-1": maspEnabled && shieldingRewardsEnabled,
+              })}
             >
-              <ShieldedRewardsBox
-                shieldedRewardsAmount={shieldedRewards.amount}
+              <NamBalanceListItem
+                className={clsx({ "h-full": !isUnbondingOrWithdrawable })}
+                currencyClassName={clsx({
+                  "block mt-4 pt-2": !isUnbondingOrWithdrawable,
+                })}
+                title="Transparent NAM"
+                color={colors.balance}
+                amount={availableAmount}
+                isLoading={isLoading}
+                isSyncing={balanceQuery.isFetching}
+              />
+              <NamBalanceListItem
+                className={clsx({ "h-full": !isUnbondingOrWithdrawable })}
+                currencyClassName={clsx({
+                  "block mt-4 pt-2": !isUnbondingOrWithdrawable,
+                })}
+                title="Staked NAM"
+                color={colors.bond}
+                amount={bondedAmount}
+                isLoading={isLoading}
+                isSyncing={stakeQuery.isFetching}
               />
-            </ListItemContainer>
-          </Stack>
+              {isUnbondingOrWithdrawable && (
+                <NamBalanceListItem
+                  title="Unbonded NAM"
+                  color={colors.unbond}
+                  amount={unbondedAmount.plus(withdrawableAmount)}
+                  isLoading={isLoading}
+                  isSyncing={stakeQuery.isFetching}
+                />
+              )}
+            </Stack>
+            <Stack className="w-full" gap={2}>
+              {isUnbondingOrWithdrawable ?
+                <>
+                  <NamBalanceListItem
+                    title="Shielded NAM"
+                    color={colors.shielded}
+                    amount={maspEnabled ? shieldedNamAmount : new BigNumber(0)}
+                    isLoading={shieldedAmountQuery.isLoading}
+                    isSyncing={isShieldSyncing}
+                    isEnabled={maspEnabled}
+                  />
+                  <ListItemContainer
+                    isEnabled={shieldingRewardsEnabled}
+                    className="flex flex-1"
+                  >
+                    <ShieldedRewardsBox
+                      shieldedRewardsAmount={shieldedRewards.amount}
+                    />
+                  </ListItemContainer>
+                </>
+              : <>
```

Comment:
@neocybereth this looks like duplicated, is it only to add the `ListItemContainer` or is there another reason? If it's only the wrapper, we have some flexibility on the layout to preserve the best code we believe. Sometimes the layout screenshot will not be 100% pixel perfect. If this is case, can you remove this condition and apply the same layout for both views?

![Screenshot 2025-03-12 at 11 22 59](https://github.com/user-attachments/assets/da118466-9d21-468e-9c18-5d827cb6deb3)

Reply:
Awesome good to know 👍 ...updated.